dokka-runners/dokka-gradle-plugin/src/main/kotlin/tasks/LogHtmlPublicationLinkTask.kt

```diff
@@ -176,22 +175,23 @@ constructor(
             }
         }
 
-//        private fun httpGetStatusJdk11(uri: URI): Result<Int> {
-//            try {
-//                val client = java.net.http.HttpClient.newHttpClient()
-//                val request = java.net.http.HttpRequest
-//                    .newBuilder()
-//                    .uri(uri)
-//                    .timeout(Duration.ofSeconds(1))
-//                    .GET()
-//                    .build()
-//                // don't care about the status - only if the server is available
-//                val response = client.send(request, java.net.http.HttpResponse.BodyHandlers.ofString())
-//                return Result.success(response.statusCode())
-//            } catch (ex: Exception) {
-//                return Result.failure(ex)
-//            }
-//        }
+        // When the minimum supported Java version is 11+
```

Comment:
never? :)

Reply:
Well, sooner than never!

https://github.com/gradle/gradle/issues/29507

So whenever we drop support for Gradle 8.